move/committee/sources/committee.move

```diff
@@ -0,0 +1,345 @@
+// Copyright (c), Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// Implementation of committee based key server operations. The admin that initializes the
+/// committee should deploy this package itself, so that the committee can manage its own upgrade
+/// and the key rotation. The key server object is owned by the committee.
+
+module seal_committee::seal_committee;
+
+use seal_testnet::key_server::{
+    KeyServer,
+    create_partial_key_server,
+    create_committee_v2,
+    PartialKeyServer
+};
+use std::{option::some, string::String};
+use sui::{transfer::Receiving, vec_map::{Self, VecMap}, vec_set::{Self, VecSet}};
+
+// ===== Errors =====
+
+const ENotMember: u64 = 0;
+const EDuplicateMember: u64 = 1;
+const EInvalidThreshold: u64 = 2;
+const EAlreadyProposed: u64 = 3;
+const EInvalidProposal: u64 = 4;
+const EAlreadyRegistered: u64 = 5;
+const EInvalidState: u64 = 6;
+const EInsufficientApprovals: u64 = 7;
+const EInsufficientOldMembers: u64 = 8;
+const ENotRegistered: u64 = 9;
+
+// ===== Structs =====
+
+/// Candidate data for a member to register with two public keys and the key server URL.
+public struct CandidateData has copy, drop, store {
```

Comment:
Maybe `MemberInfo`?
we call those members anywhere & candidate is a state of a committee

Reply:
edit: party_id is particular to the dkg protocol in fastcrypto. going to keep it unchanged to be consistent